style.css

```diff
@@ -468,3 +502,242 @@ body {
+    line-height: 28px;
+    color: #344563;
+    padding: 0;
+  }
+
+  .connect {
+    padding: 0;
+  }
+
+  .connect h3 {
+    display: flex;
+    flex-direction: row;
+    align-items: start;
+    justify-content: left;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .logos ul {
+    display: flex;
+    gap: 20px;
+    color: #505f79;
+    list-style: none;
+    margin-left: 0.1rem;
+    margin-top: 0.5rem;
+  }
+
+  .break {
+    display: none;
+  }
+
+  .heading-menu {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    text-align: center;
+    gap: 32px;
+    font-family: 'Poppins', sans-serif;
+    color: #634634;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-right: 155px;
+  }
+
+  .menubars {
+    display: none;
+  }
+
+  .cards {
+    display: grid;
+    padding: 16px;
+    grid-template-columns: repeat(2, 1fr);
+    justify-content: center;
+    place-items: center;
+    gap: 20px;
+    background-color: #e5e5e5;
+  }
+}
+
+@media screen and (min-width: 1280px) {
+  /* header {
+    margin: 1.25rem 2.5rem;
+  } */
+
+  .logo {
+    margin-left: 155px;
+  }
+
+  .cards {
+    grid-template-columns: repeat(1, 1fr);
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 16px;
+    gap: 46px;
+    height: 496px;
+    width: 1156px;
+    border-radius: 24px;
+    margin: 60px 0;
+  }
+
+  .card:nth-child(even) {
+    flex-direction: row-reverse;
+    padding: 48px;
+  }
+
+  .card-image img {
+    height: 448px;
+    width: 544px;
+    border-radius: 8px;
+    padding: 24px;
+  }
+
+  .card-text h3 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .card-text h3 span {
+    display: inline;
+  }
+
+  .BSimage {
+    display: block;
+  }
+
+  .SSimage {
+    display: none;
+  }
+
+  .about-me {
+    display: flex;
+    flex-direction: row;
+    padding: 166px 182px 166px 143px;
+  }
+
+  .bigscreen {
+    width: 447px;
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+  }
+
+  .more-details {
+    width: 608px;
+    margin: -30px 0 0 60px;
+  }
+
+  .language ul li {
+    flex-direction: column;
+    display: flex;
+    align-items: center;
+    width: 122px;
+    height: 120px;
+    background: #f7f7f9;
+    border-radius: 8px;
+    margin: 0 0 0 -24px;
+    padding: 0 24px;
+    gap: 16px;
+    justify-content: center;
+  }
+
+  .language ul {
+    display: flex;
+    flex-direction: row;
+    gap: 36px;
+  }
+
+  .form-section {
+    background-image: url(./images/Aboutme-Big-screen.png);
+    background-position: center;
+    background-size: 80%;
+    background-repeat: no-repeat;
+    background-color: #4949de;
+    justify-content: center;
+    align-items: center;
```

Comment:
- [ ] (OPTIONAL) Please apply the right background image as it is overlapping the text and makes it hard to read:
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/91486791/79c8d0e8-9139-4371-ac7e-a191bfdf2434)
figma:
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/91486791/ea5989c9-ee1c-4997-8ed6-469cd36855ad)